src/lib/support/nlargparser.hpp

```diff
@@ -70,6 +70,11 @@ struct OptionDef
                                              for the option. */
 };
 
+/**
+ * Defines a terminal sentinel for a sequence of OptionDef structs.
+ */
+const OptionDef OptionDefEndOfList = { nullptr, kNoArgument, 0 };
```

Comment:
Minor nit, but I might suggest calling this "kOptionDefEndOfList" to indicate the constant convention.

Reply:
Done, thanks :)